diff --git a/schema/xsd/thirdparty/XAdES.xsd b/schema/xsd/thirdparty/XAdES.xsd
index 3c04fbe..6a1973f 100644
--- a/schema/xsd/thirdparty/XAdES.xsd
+++ b/schema/xsd/thirdparty/XAdES.xsd
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xsd:schema targetNamespace="http://uri.etsi.org/01903/v1.3.2#" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://uri.etsi.org/01903/v1.3.2#" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" elementFormDefault="qualified">
-	<xsd:import namespace="http://www.w3.org/2000/09/xmldsig#"/>
+	<xsd:import schemaLocation="xmldsig-core-schema.xsd" namespace="http://www.w3.org/2000/09/xmldsig#"/>
 	<!-- Start auxiliary types definitions: AnyType, ObjectIdentifierType,
 EncapsulatedPKIDataType and containers for time-stamp tokens -->
 	<!-- Start AnyType -->
